pages/linux/pvs.md

```diff
@@ -0,0 +1,31 @@
+# pvs
+
+> Display information about physical volumes.
```

Comment:
Hey, cool page! I was confused by the description at first though when I tried it for myself, before I looked it up and realised this is to do LVM. Perhaps we could include that here?

```suggestion
> Display information about LVM physical volumes.
```

Reply:
That's totally justified, I'll soon make a PR adding this.